manifests/profile/pdf_stack.pp

```diff
@@ -22,9 +22,16 @@
     require             => Yumrepo['robert-gcj'],
   }
 
-  package { ['wkhtmltopdf', 'pdftk']:
+  package { 'pdftk':
     ensure  => present,
-    require => Yumrepo['robert-pdftk'],
+    require => [Yumrepo['robert-pdftk'],Package[$fonts]],
+  }
+
+  package { 'wkhtmltopdf':
+    ensure   => present,
+    source   => 'https://s3-us-west-2.amazonaws.com/education-packages/wkhtmltopdf-1.0-1.x86_64.rpm',
```

Comment:
Can we use the accurate version number here? Maybe something like `0.12.3-puppet_education`?